docs/docs-content/clusters/data-center/nutanix/nutanix.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_label: "Nutanix"
+title: "Nutanix"
+description: "Learn how to configure Nutanix and create Nutanix clusters in Palette."
+hide_table_of_contents: false
+sidebar_position: 10
+tags: ["data center", "nutanix"]
+---
+
+
+[Nutanix](https://www.nutanix.com/what-we-do) is a hyper-converged infrastructure platform that combines storage, compute, and networking into a single integrated system. Palette provides a generic framework built upon the open-source [Cluster API (CAPI)](https://cluster-api.sigs.k8s.io) initiative to support adding cloud providers. As a Technical Preview of implementing this framework, you can deploy Kubernetes clusters on Nutanix using Palette.  Integration with Nutanix is achieved through a Private Cloud Gateway (PCG), which establishes a secure connection with Nutanix.
+
+## Get Started
+
+Once a system administrator registers Nutanix with Palette, the PCG is installed, and your Nutanix cloud account is added, you can deploy a cluster to Nutanix by using Palette. Check out the [Deploy a Cluster with Palette](../../public-cloud/deploy-k8s-cluster.md) tutorial to get started.
```

Comment:
The semantic of the sentence is a bit confusing: I had to read it 3 times to understand that those are requirements and the PCG is not (automatically) installed after "...a system administrator registers Nutanix with Palette"